### ISSUE TITLE
Clarify that ACT Framework Requirements is not about ACT Rules

### ISSUE THREAD

**Issue by rdeltour** [w3c/wcag-act#6]
We discussed that already, but apparently it's still not clear enough, as @michael-n-cooper commented on the survey:

> Can't reconcile "ACT Rules should map to [WCAG 2.0 Failure Techniques]" and "It does not include any ACT Rules". Either it maps rules or it doesn't include rules, not both.
>
> Overall this seems like both requirements for the ACT Framework, and draft content for the ACT Framework. Makes it hard to comment, those need to be teased out.

We need to make another pass on that, in particular to ensure that we don't have any RFC 2119 statements about ACT Rules.

**Comment by WilcoFiers** [comment]
This should be resolved together with https://github.com/w3c/wcag-act/issues/7

**Comment by moekraft** [comment]
We no longer are following a 1-1 mapping between techniques and rules. Wilco to update.

**Comment by WilcoFiers** [comment]
Oudated, closing issue.
